app/src/androidTest/java/org/simple/clinic/user/UserDaoAndroidTest.kt

```diff
@@ -0,0 +1,50 @@
+package org.simple.clinic.user
+
+import android.content.SharedPreferences
+import android.support.test.runner.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.simple.clinic.AppDatabase
+import org.simple.clinic.PatientFaker
+import org.simple.clinic.TestClinicApp
+import javax.inject.Inject
+
+@RunWith(AndroidJUnit4::class)
+class UserDaoAndroidTest {
+
+  @Inject
+  lateinit var sharedPrefs: SharedPreferences
+
+  @Inject
+  lateinit var appDatabase: AppDatabase
+
+  @Inject
+  lateinit var patientFaker: PatientFaker
+
+  @Before
+  fun setup() {
+    TestClinicApp.appComponent().inject(this)
+
+    appDatabase.clearAllTables()
+    sharedPrefs.edit().clear().apply()
+  }
+
+  /**
```

Comment:
I think the type of comments used always depends on where you're using them. So function docs always have asterisks. Comments inside functions have `//`. 

https://source.android.com/setup/contribute/code-style#use-javadoc-standard-comments

Reply:
Right, but this method doesn't explain what the function does; it explains why it was added. To me, this is more of a multi-line comment instead of a doc.

Anyway, not important right now.

Reply:
🙂👍